apps/golden-sample-app/src/app/theme-switcher/theme-switcher.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from "@angular/core";
+import { ThemeService } from "./theme-service";
+
+@Component({
+selector: 'app-theme-switcher',
+templateUrl: './theme-switcher.component.html',
+})
+export class ThemeExporterComponent{
+  @Input()
+  theme = ThemeService.THEMES[0];
+
+  themes = ThemeService.THEMES;
```

Comment:
Is this used?

Reply:
`themes` is being consumed by `[options]` of the `dropdown-menu-ui` in the HTML